suites/apis/centralizedAuth.js

```diff
@@ -0,0 +1,639 @@
+  await indexd.do.deleteFileIndices(Object.values(indexed_files));
+
+  console.log('Adding indexd files used to test signed urls');
+  const ok = await indexd.do.addFileIndices(Object.values(indexed_files));
+  chai.expect(
+    ok, 'unable to add files to indexd as part of centralizedAuth setup').to.be.true;
+});
+
+AfterSuite(async (fence, indexd, users) => {
+  console.log('Removing indexd files used to test signed urls');
+  await indexd.do.deleteFileIndices(Object.values(indexed_files));
+});
+
+Before(async (fence, users, indexd) => {
+  // populate new GUIDs per test
+  const gen3FooBarFileGuid = uuid.v4().toString();
+  const gen3DeleteMe = uuid.v4().toString();
+  const abcFooBarFileGuid = uuid.v4().toString();
+  const abcDeleteMe = uuid.v4().toString();
+
+  new_gen3_records.fooBarFile.did = gen3FooBarFileGuid;
+  new_gen3_records.deleteMe.did = gen3DeleteMe;
+  new_abc_records.fooBarFile.did = abcFooBarFileGuid;
+  new_abc_records.deleteMe.did = abcDeleteMe;
+});
+
+After(async (fence, users, indexd) => {
+  // Cleanup after each scenario
+  console.log('Removing test indexd records if they exist');
+  await indexd.do.deleteFileIndices(Object.values(new_gen3_records));
+  await indexd.do.deleteFileIndices(Object.values(new_abc_records));
+});
+
+/*
+   - Test that user without policies can't CRUD records in /gen3 or /abc
+*/
+Scenario('User without policies cannot CRUD indexd records in /gen3 or /abc @indexdJWT',
+  async (fence, indexd, users, files) => {
+    // create
+    const gen3_create_success = await indexd.do.addFileIndices(
+        Object.values(new_gen3_records), users.user2.accessTokenHeader);
+    const abc_create_success = await indexd.do.addFileIndices(
+        Object.values(new_abc_records), users.user2.accessTokenHeader);
+
+    // asserts for create
+    chai.expect(
+      gen3_create_success,
+      'uh oh, user without permission was able to register files to indexd under `/gen3`'
+    ).to.be.false;
+    chai.expect(
+      abc_create_success,
+      'uh oh, user without permission was able to register files to indexd under `/abc`'
+    ).to.be.false;
+
+    // read
+    const gen3_read_response = indexd.do.getFile(
+        new_gen3_records.fooBarFile, users.user2.accessTokenHeader)
+    const abc_read_response = indexd.do.getFile(
+        new_abc_records.fooBarFile, users.user2.accessTokenHeader)
+
+    // asserts for read
+    fenceQuestions.assertStatusCode(
+        gen3_read_response, 403,
+        msg='should have gotten unauthorized for reading record under `/gen3`'
+    );
+    fenceQuestions.assertStatusCode(
+        abc_read_response, 403,
+        msg='should have gotten unauthorized for reading record under `/abc`'
+    );
+
+    // update
+    const filename_change = {
+      'file_name': 'test_filename'
+    }
+    const gen3_update_response = indexd.do.updateFile(
+        new_gen3_records.fooBarFile, filename_change, users.user2.accessTokenHeader)
```

Comment:
btw i am curious/confused about why these tests passed when you ran them with `accessTokenHeader`

Reply:
:thinking: hmm... I will need to investigate this one. Now that you point it out, I am also confused

Reply:
You haven't changed this, did you figure out what's up with the headers?

Reply:
various indexd updates to support this now, so this should work passing through user tokens as expected

Reply:
oh, cool! then can you change the default parameter in `addFileIndices`, `updateFile` and `deleteFile` to be `accessTokenHeader` instead of `indexdAuthHeader`?

Reply:
I think it would make other tests fail. `cdis.autotest@gmail.com` only has admin privileges in `abc` namespace for testing. So any other test trying to do other things would fail. I support I could make an additional admin user, but I figured we could wait to do that until we truly deprecate the basic auth that the current header is using